client/packages/requisitions/src/RequestRequisition/api/operations.generated.ts

```diff
@@ -5,719 +5,1549 @@ import gql from 'graphql-tag';
 import { ReasonOptionRowFragmentDoc } from '../../../../system/src/ReasonOption/api/operations.generated';
 import { NameRowFragmentDoc } from '../../../../system/src/Name/api/operations.generated';
 type GraphQLClientRequestHeaders = RequestOptions['requestHeaders'];
-export type ItemWithStatsFragment = { __typename: 'ItemNode', id: string, name: string, code: string, unitName?: string | null, defaultPackSize: number, availableStockOnHand: number, stats: { __typename: 'ItemStatsNode', averageMonthlyConsumption: number, availableStockOnHand: number, totalStockOnHand: number, availableMonthsOfStockOnHand?: number | null, totalConsumption: number } };
+export type ItemWithStatsFragment = {
```

Comment:
@fergie-nz these files should not be saved with formatting, otherwise we would be getting big diffs